backend/supabase/user_contract_metrics/triggers.sql

```diff
@@ -0,0 +1,38 @@
+alter table user_contract_metrics
+add column has_yes_shares boolean,
+add column has_no_shares boolean,
+add column total_shares_yes numeric,
+add column total_shares_no numeric,
+add column profit numeric,
+add column has_shares boolean;
```

Comment:
We should come to agree on what to commit in sql. This approach is kind of a like a migration script of what you ran once.

I think the previous idea is that all the files could be run and produce the current state of the db. For example, when I add a column, I run the alter table command, but then only commit an edit to the initial creation of the table in `seed.sql` so that it represents the current schema.

For now this is fine, but I think we should pick a style and stick to it.